docs/README.md

```diff
@@ -1,65 +1,28 @@
-## General
 
-### What is Chainsfer?
+# Introduction to Chainsfer.io
 
-Chainsfer is a cryptocurrency transfer service, powered by Google Drive APIs. 
-Chainsfer is a service developed and maintained by Ventureum Inc.
-Ventureum Inc. is a registered Canadian company based in Toronto, Ontario.
+### What is Chainsfer? 
+* Chainsfer is a cryptocurrency transfer service, powered by Google Drive APIs.
+* Chainsfer is a service developed and maintained by Ventureum Inc.
+* Ventureum Inc. is a registered Canadian company based in Toronto, Ontario.
 
 ### Why choose Chainsfer?
-
 Chainsfer aims to make cryptocurrency transaction more friendly and fault-tolerant, easy to use.
 
-### How does Chainsfer work and how safe is Chainsfer?
-
-As a financial and cryptocurrency service, we try our best to build Chainsfer as secure as possible.
-
-Chainsfer service use your Google account to store your crypto wallet information. 
-Learn more about the Google third-party, please check the links below: 
-Allow 3rd-party apps for Drive files
-Sharing your Google data with Apps
-
 ### Is Chainsfer free?
-
 Yes. The Chainsfer service itself does not charge any service fee. However you still have to pay gas fee on the blockchain.
 
-### Do I need to register for an Chainsfer account to send or receive?
+### How does Chainsfer work and how safe is Chainsfer?
+As a financial and cryptocurrency service, we try our best to build Chainsfer as secure as possible. Chainsfer service use your Google account to store your crypto wallet information. 
+> Learn more about the Google third-party, please check the links below: 
+> * [Allow 3rd-party apps for Drive files](https://support.google.com/a/answer/6105699?hl=en)
+> * [Sharing your Google data with Apps](https://www.youtube.com/watch?time_continue=4&v=W1a1lQHVtJo)
 
-No. 
-You only need a Google account. You can track the update of your transaction through the link in your email.
-However, we high recommend you to register a Chainfer account, which will give you better user experience.
+### Do I need to register for an Chainsfer account to send or receive?
+No. You only need a Google account. You can track the update of your transaction through the link in your email. However, __we high recommend you to register a Chainfer account__, which will give you better user experience.
```

Comment:
"You can track the status of your transaction", or "get updates regarding your tx through ...."